libs/atqr/domain/src/lib/services/payment/interfaces/payment.interface.ts

```diff
@@ -0,0 +1,15 @@
+import { PaymentMethodEntity } from '../../../PaymentMethod';
+
+export interface IPaymentService {
+  createCostumer(costumerId: string): void;
+  createPaymentMethod(
+    costumerId: string,
+    paymentMethod: PaymentMethodEntity
+  ): void;
+  setupFuturePayment(
```

Comment:
This function should return a FuturePaymentId and should be updated when it is possible.